src/config/datasource_config.py

```diff
@@ -22,11 +22,11 @@ def build_database_connection_string():
 connect_args = {"check_same_thread": False}
 engine = create_engine(SQLALCHEMY_DATABASE_URL, echo=True)
 
-def create_db_and_table():
+async def create_db_and_table():
     print("creating sql tables")
```

Comment:
I think we should use logging library instead of print statement

Reply:
The logging i am trying to achieve will most likely be a traceable logger.
so yes, we'll be using a logging library.|
This is just a placeholder for now.

Logging will be in a different PR.